### PR TITLE
made error list to report Project ID Guid to error list

### PR DESCRIPTION
**Pull request by heejaechang**
we decide to move away from IVsHierarchy and instead use Project Id Guid. we just need some kind of ID to distinguish between different projects. IVsHierarchy is just not right one to use for that purpose.

### CODE REVIEW

**Comment by heejaechang** [comment]
@srivatsn @shyamnamboodiripad @mavasani can you take a look please?

**Comment by heejaechang** [comment]
fix  #1859

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs [hunk 1]
```diff
@@ -212,7 +217,8 @@ protected string GetProjectName(Workspace workspace, ProjectId projectId)
             return project.Name;
         }
 
-        protected IVsHierarchy GetHierarchy(Workspace workspace, ProjectId projectId, DocumentId documentId)
+        // TODO: remove this once we moved to new drop
```
Comment (mavasani) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs]: Open a github issue for the TODOs?

**Comment by mavasani** [comment]
:+1:

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs (hunk 1 above)
Comment (heejaechang) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs]: sure #2400 opened.

**Comment by srivatsn** [comment]
:+1:

**Pull request merged**